package.json

```diff
@@ -36,8 +36,13 @@
     "pegjs": "0.7.0",
     "request": "2.79.0",
     "underscore": "1.8.3",
-    "winston": "2.3.0",
-    "nodeunit": "0.10.2",
+    "winston": "2.3.0"
+  },
+  "devDependencies": {
```

Comment:
I don't even think I was thinking about ``mdn-browser-compat-data`` when you mentioned your concern. I think it just got caught up in the split of "devDependencies" from "dependencies" since it was the last in the list. Another case of me looking but not seeing! 😄